docs/workforce-management-framework/topics/con-introducing-securityfeatures.adoc

```diff
@@ -2,11 +2,11 @@
 = Introducing {WFM-RC-NameShort} Security Features
 
 {WFM-RC-NameShort} Security is built around a *Security Interface* which abstracts itself from all {WFM-RC-NameShort} security libraries.
-{WFM-RC-NameShort} allows you to build a custom security solution for integration with {WFM-RC-NameShort}.
-{WFM-RC-NameShort} also provides two security implementations out-of-the-box and are called link:http://www.keycloak.org/[Keycloak] and link:http://passportjs.org/[Passport.js].
+{WFM-RC-NameShort} allows you to integrate custom security solution.
+{WFM-RC-NameShort} also provides two out-of-the-box security implementations: link:http://www.keycloak.org/[Keycloak] and link:http://passportjs.org/[Passport.js].
 
-*Keycloak* and *Passport.js* both provide authentication and authorization on the server side.
-*Keycloak* and *Passport.js* implementations can replace each other without breaking the internal api belonging to all {WFM-RC-NameShort} module's and also any of the reference applications.
+*Keycloak* and *Passport.js* both provide server side authentication and authorization.
+*Keycloak* and *Passport.js* implementations can replace each other without breaking the internal {WFM-RC-NameShort} api, modules and also any of the reference applications.
```

Comment:
This change has a different meaning than the previous text.

If it still holds true from a technical POV - I'm a +1 as it reads better.

Reply:
@wtrocki are you ok with this change?